sdk/storage/azure-storage-queue/tests/test_queue.py

```diff
@@ -1349,6 +1349,67 @@ def test_transport_closed_only_once(self, **kwargs):
             qsc.get_service_properties()
             assert transport.session is not None
 
+    @QueuePreparer()
+    @recorded_by_proxy
+    def test_public_audience_queue_client(self, **kwargs):
```

Comment:
Probably don't need a test for public anymore since its just specifying nothing. We already test this elsewhere.